### DISCUSSION TITLE
Responsibe DataTable

### DISCUSSION THREAD

**Discussion by ftorreslop** [primefaces/community#806] [PrimeVue] [answered]
Is there a way to make a datatable responsive, putting header and value below and the same below?.

**Comment by tugcekucukoglu** [comment, marked as answer]
I cannot clearly understand what you mean. In anyway there are `header` and `body` slots for Column component.

**Comment by Tamas-hi** [comment]
I think the author means that DataTable has deprecated the 'stack' layout as of PrimeVue 3.24.0 (see details: https://github.com/primefaces/primevue/issues/4131)

We would really appreciate a mobile friendly UI support (like the 'stack' layout before). Thank you.
